### DISCUSSION TITLE
What does "network" mean in "List public events for a network of repositories"?

### DISCUSSION THREAD

**Discussion by tisonkun** [community/community#24757] [Apps, API and Webhooks]
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/rest/reference/activity#list-public-events-for-a-network-of-repositories" target="_blank" rel="noopener nofollow ugc">docs.github.com</a>
  </header>
  <article class="onebox-body">
    

<h3><a href="https://docs.github.com/en/rest/reference/activity#list-public-events-for-a-network-of-repositories" target="_blank" rel="noopener nofollow ugc">Activity - GitHub Docs</a></h3>

<p><a href="https://docs.github.com/en/rest/reference/activity#list-public-events-for-a-network-of-repositories" target="_blank" rel="noopener nofollow ugc">//docs.github.com/en/rest/reference/activity</a></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by byrneh** [comment, marked as answer]
<p>Hi <a class="mention" href="/tisonkun">@tisonkun</a>,</p>
<p>I would guess it relates to the repositories network, there is documentation reference here about  <a href="https://docs.github.com/en/github/visualizing-repository-data-with-graphs/viewing-a-repositorys-network" rel="noopener nofollow ugc">viewing-a-repositorys-network</a>. GitHub maintains a network for example of forked repositories from the parent.</p>

**Comment by tisonkun** [comment]
<p>Thanks! That sounds reasonable.</p>
